rpcs/ledgerService_test.go

```diff
@@ -53,6 +54,27 @@ type httpTestPeerSource struct {
 func (s *httpTestPeerSource) GetPeers(options ...network.PeerOption) []network.Peer {
 	return s.peers
 }
+func (s *httpTestPeerSource) Address() (string, bool) {
+	return "", false
+}
+func (s *httpTestPeerSource) Broadcast(ctx context.Context, tag protocol.Tag, data []byte, wait bool, except network.Peer) error {
+	return errors.New("mock MockClientAggregator does not implement Broadcast")
```

Comment:
If you already decided to have your own custom errors, you should at least make sure these would contain a meaning text..